frontend/src/services/auth/UserProvider.tsx

```diff
@@ -85,11 +85,14 @@ export default function UserProvider({
   const refreshIfExpired = useCallback(async (): Promise<
     boolean | undefined
   > => {
+    if (!localUser?.token) {
```

Comment:
since logging out the local user does not update the expiration date, making this check helps to ensure that the function won't run if the user is logged out